chain/actors/policy/policy.go

```diff
@@ -527,6 +527,12 @@ func GetMaxSectorExpirationExtension() abi.ChainEpoch {
 	return miner11.MaxSectorExpirationExtension
 }
 
+func GetDefaultSectorExpirationExtension() abi.ChainEpoch {
```

Comment:
I'm not a huge fan of adding this here, because it's _not_ actors policy (it's just the default we wanna use in Lotus).

But I see why it's the easiest thing right now.